### PR TITLE
Distinguish MDK from source distribution

### PR DESCRIPTION
**Pull request by jpaugh**
Also added to the latest branch (1.16.x). See PR #350

### CODE REVIEW

File: docs/gettingstarted/index.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 From Zero to Modding
 --------------------
 
-1. Obtain a source distribution from forge's [files][] site. (Look for the MDK file type)
-2. Extract the downloaded source distribution to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
+1. Obtain the MCForge Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
```
Comment (LexManos) [inline docs/gettingstarted/index.md]: Mod Development Kit

Comment (jpaugh) [reply docs/gettingstarted/index.md]: You're right! That reads way better! I've updated both PRs to fix this.

File: docs/gettingstarted/index.md [hunk 2]
```diff
@@ -6,8 +6,8 @@ This is a simple guide to get you from nothing to a basic mod. The rest of this
 From Zero to Modding
 --------------------
 
-1. Obtain a source distribution from forge's [files][] site. (Look for the MDK file type)
-2. Extract the downloaded source distribution to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
+1. Obtain the Mod Development Kit (MDK) from forge's [files][] site. (Look for the MDK file type)
+2. Extract the downloaded MDK an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
```
Comment (sciwhiz12) [inline docs/gettingstarted/index.md]: Missing a `to` there.
```suggestion
2. Extract the downloaded MDK to an empty directory. You should see a bunch of files, and an example mod is placed in `src/main/java` for you to look at. Only a few of these files are strictly necessary for mod development, and you may reuse these files for all your projects These files are:
```

**Review by sciwhiz12 — CHANGES_REQUESTED** [review]
One small minor correction to the grammar, then it looks good to me.

**Review by TheCurle — APPROVED** [review]
I wonder how long this has been wrong.

**Pull request merged**